libs/compute_api/src/models.rs

```diff
@@ -0,0 +1,77 @@
+//! Structs representing the JSON formats used in the compute_ctl's HTTP API.
+use chrono::{DateTime, Utc};
+use serde::{Deserialize, Serialize, Serializer};
+
+use crate::spec::ComputeSpec;
+
+/// Request of the /configure API
+///
+/// We now pass only `spec` in the configuration request, but later we can
+/// extend it and something like `restart: bool` or something else. So put
+/// `spec` into a struct initially to be more flexible in the future.
+#[derive(Deserialize, Debug)]
+pub struct ConfigurationRequest {
+    pub spec: ComputeSpec,
+}
```

Comment:
What do you think about separating http request/responses into different files as it's in the main now? It makes usage and navigation more clean

We can keep common parts in the `models.rs`, though

Reply:
Ok, I split it up again. There's nothing but ConfigurationRequest in requests.rs now, but maybe we'll grow more.